src/main/java/com/salesforce/mirus/metrics/MirrorJmxReporter.java

```diff
@@ -15,11 +17,25 @@
 
   private static final Logger logger = LoggerFactory.getLogger(MirrorJmxReporter.class);
 
+  public static Map<Integer, String> LATENCY_BUCKETS =
+      Map.of(
+          0,
+          "0m",
+          5 * 60 * 1000,
+          "5m",
+          10 * 60 * 1000,
+          "10m",
+          30 * 60 * 1000,
+          "30m",
+          60 * 60 * 1000,
+          "60m");
```

Comment:
Before we commit to the buckets we should discuss. We definitely need 0,5,10.  Perhaps we should add one more for very old records, maybe 12 or 24 hours?